ansible/server-state-playbooks/www-dev/Vagrantfile

```diff
@@ -0,0 +1,15 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "centos/7"
+  config.vm.provider "virtualbox" do |vb|
+  config.vm.network "forwarded_port", guest: 80, host: 8080
+  config.vm.network "forwarded_port", guest: 4064, host: 4064
```

Comment:
Addressed in https://github.com/openmicroscopy/infrastructure/pull/281/commits/c3845609de90d79fb08b2b3858e63b3e1cfcf811